.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: bionic
```

Comment:
Why is this change?

Reply:
That is Ubuntu 18.04 LTS, which is latest and greatest :)

Reply:
Just thought it is nice to use the latest though.